src/app/page.tsx

```diff
@@ -110,17 +108,21 @@ function JurisdictionSection() {
       </div>
     </>
   );
-}
+});
 
-function InvitationCta() {
+const InvitationCta = memo(function InvitationCta() {
   return (
     <section className="usa-graphic-list usa-section usa-section--light-blue">
       <GridContainer>
         <div className="flex flex-col items-center justify-center gap-5 self-stretch">
-          <div className="self-stretch text-center text-[1.5rem] font-bold text-[#224a58] xl:text-[2rem]">
+          <div
```

Comment:
Doesn't look like you switched this, but should this be an `h2`?

Reply:
yes it should